README.md

```diff
@@ -1 +1,72 @@
-# Group-Capstone-
\ No newline at end of file
+# Screenshot
+[screenshot](src/groupCapstone.png) 
```

Comment:
- [ ] Please update the screenshots section of the README the current path is a `link` not an `image`.